pkg/server/server.go

```diff
@@ -0,0 +1,36 @@
+package server
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/marekparafianowicz/go-server/sites"
+)
+
+type crawler interface {
+	GetPage() error
+}
+
+type server struct {
+	c crawler
+}
+
+func New(c crawler) server {
+	return server{c}
+}
+
+func (s server) Run(port int) error {
+	router := gin.Default()
+
+	router.GET("/", s.c.GetPage())
```

Comment:
To ofc nie działa, zastanawiam się jak zrobić, żeby w tym pliku nie mieć logiki `GetPage`, jak to zaimportować nie używając `import "github.com/marekparafianowicz/go-server/pages"`